src/simulator_core/simulation/networksimulation.py

```diff
@@ -61,3 +62,7 @@ def gather_output(self) -> DataFrame:
         """
         result = self.network.gather_output()
         return result
+
+    def plot_output(self) -> None:
+        """Plot the result for inspections."""
```

Comment:
Maybe expand the description what does it plot; where does it get the data from?